frontend/src/components/tables/Columns/Users/UserColumns.tsx

```diff
@@ -38,7 +38,7 @@ export const columns: ColumnDef<IUser>[] = [
   },
 
   {
-    accessorKey: "name",
+    accessorKey: "firstName",
```

Comment:
add an id to make the name in the filter show שם and not FirstName. You can do it with other columns that have weird looking names in the filter.